osler/inventory/tests/test_views.py

```diff
@@ -153,3 +155,41 @@ def test_drug_add_new(self):
                              str(getattr(new_drug, param)))
 
 
+class TestDrugExport(TestCase):
+    """
+    Tests csv exports with correct name and user permissions work
+    """
+    def setUp(self):
+        log_in_user(self.client, build_user())
+
+    def test_export_csv(self):
+        export_csv_url = reverse('inventory:export-csv')
+        no_perm_group = user_factories.NoPermGroupFactory()
+        csv_perm_group = user_factories.PermGroupFactory(permissions=['inventory.export_csv'])
+
+        for group in [no_perm_group, csv_perm_group]:
+            log_in_user(self.client, user_factories.UserFactory(groups=[group]))
+            response = self.client.post(export_csv_url)
+
+            if group == csv_perm_group:
+                assert response.status_code == 200
+                self.assertEqual(response["Content-Disposition"],
+                                 ''.join(['attachment; filename=drug-inventory-', str(date.today()), '.csv']))
+            else:
+                 assert response.status_code == 403
```

Comment:
It looks like you're testing for permission here, which is very good. I was also wondering if you had testing for the actual drug (like 1 drug for example) being exported.

Reply:
Oh my bad, I think you actually have it on line 176.